sdk/src/main/java/io/opentdf/platform/sdk/TDF.java

```diff
@@ -35,9 +37,10 @@
  */
 public class TDF {
 
+    public static final byte[] GLOBAL_KEY_SALT = "salt".getBytes(StandardCharsets.UTF_8);
     private static final String EMPTY_SPLIT_ID = "";
     private static final String TDF_VERSION = "4.3.0";
-    private static final String KEY_ACCESS_SECHMA_VERSION = "4.3.0";
+    private static final String KEY_ACCESS_SECHMA_VERSION = "1.0";
```

Comment:
Not sure about this, is it intended?

Reply:
In Go sdk we have it as 1.0 - making it consistent